lib/pages/account/select_theme.dart

```diff
@@ -0,0 +1,155 @@
+            child: ListView.separated(
+              itemCount: themeList.length,
+              itemBuilder: (context, index) {
+                return themeTile(
+                  theme: themeList[index],
+                  index: index,
+                  length: themeList.length,
+                );
+              },
+              separatorBuilder: (context, index) => Divider(
+                color: Theme.of(context).colorScheme.secondaryVariant,
+                thickness: 0.3,
+                height: 2,
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget themeTile({
+    required String theme,
+    required int index,
+    required int length,
+  }) {
+    return GestureDetector(
+        child: Column(
+          children: [
+            Container(
+              height: 48,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.only(
+                  topLeft:
+                      index == 0 ? Radius.circular(12.0) : Radius.circular(0),
+                  topRight:
+                      index == 0 ? Radius.circular(12.0) : Radius.circular(0),
+                  bottomLeft: index == length - 1
+                      ? Radius.circular(12.0)
+                      : Radius.circular(0),
+                  bottomRight: index == length - 1
+                      ? Radius.circular(12.0)
+                      : Radius.circular(0),
+                ),
+                color: Theme.of(context).colorScheme.secondaryVariant,
+              ),
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 14.0),
+                child: Row(
+                  children: [
+                    Text(
+                      getThemeLocalizationString(theme: theme, ctx: context),
+                      style: Theme.of(context).textTheme.headline1!.copyWith(
+                          fontSize: 15, fontWeight: FontWeight.normal),
+                    ),
+                    Spacer(),
+                    BlocBuilder<ThemeCubit, ThemeState>(
+                      bloc: Get.find<ThemeCubit>(),
+                      builder: (context, state) {
+                        if (state.themeStatus == ThemeStatus.done) {
+                          if (state.theme == theme) {
+                            return Icon(
+                              Icons.check_circle_rounded,
+                              size: 26,
+                              color: Theme.of(context).colorScheme.surface,
+                            );
+                          } else {
+                            return SizedBox.shrink();
+                          }
+                        } else {
+                          return SizedBox.shrink();
+                        }
+                      },
+                    )
+                  ],
+                ),
+              ),
+            ),
+            index == length - 1
```

Comment:
pls separate this one into outside of listview, so that every items of listview can render without checking more for this.